dismantle-tablegen/dismantle-tablegen.cabal

```diff
@@ -101,4 +101,4 @@ test-suite dismantle-tests
                , text
                , tasty >= 0.10
                , tasty-hunit
-               , tasty-quickcheck >= 0.8 && <= 0.11
+               , tasty-quickcheck >= 0.8 && <= 0.12
```

Comment:
While we're at it, using `<=` for upper version bounds is pretty uncommon. I'd prefer using `< 0.13` instead of `<= 0.12`.